### PR TITLE
Mariner 3.0 - bulk upgrade of perl spec and some perl-* specs

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Mariner 3.0 - bulk upgrade of perl spec and some perl-* specs

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/perl/perl.spec [hunk 1]
```diff
@@ -533,9 +542,8 @@ packages like perldoc by perl-Pod-Perldoc and splain by perl-diagnostics.
 Summary:        A module for Perl manipulation of .tar files
 License:        GPL+ or Artistic
 Epoch:          0
```
Comment (anphel31) [inline SPECS/perl/perl.spec:544]: If possible it would be great to get rid of all of the Epochs

Comment (nicogbg) [reply SPECS/perl/perl.spec:544]: I just kept them inline with Fedora spec

Comment (nicogbg) [reply SPECS/perl/perl.spec:544]: I will not fix them in this upgrade

File: SPECS/perl-B-COW/perl-B-COW.spec [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+BuildRequires:	perl(Test::More)
+
+# Dependencies
+Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+
+# Don't "provide" private Perl libs
+%{?perl_default_filter}
+
+%description
+B::COW provides some naÃ¯ve additional B helpers to check the Copy On Write
```
[the remark this answers is not in the document]
Comment (nicogbg) [reply SPECS/perl-B-COW/perl-B-COW.spec]: ooops, fixed

File: SPECS/perl/perl.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/perl/perl.spec:544]: task filed to track this https://microsoft.visualstudio.com/OS/_workitems/edit/48377655

**Comment by anphel31** [comment]
This is a big change - please run a dev build for both architectures before merging

**Review by anphel31 — APPROVED** [review]

**Pull request merged**